src/main/java/com/redhat/cloud/notifications/routers/InternalService.java

```diff
@@ -34,10 +36,10 @@
     BundleResources bundleResources;
 
     @Inject
-    BehaviorGroupResources behaviorGroupResources;
+    ApplicationResources appResources;
```

Comment:
Why do we need to change these injections?

Reply:
Didnt we need BahaviorGroupResources?

Reply:
I removed an internal API that used `BehaviorGroupResources` recently and forgot to remove the injection. It's no longer needed here.